### PR TITLE
Upgrade eksportisto suffix 11

### LINKED ISSUE #7342: Deploy eksportisto on mainnet with cEUR support

### PR DESCRIPTION
**Pull request by tkporter**
### Description

* Upgrades eksportisto suffix 11 to use https://github.com/celo-org/eksportisto/pull/47

### Other changes

Added a headless service for the statefulset. This isn't necessarily required, which is why it didn't exist before, but [will fail validation](https://github.com/kubernetes/kubernetes/issues/69608#issuecomment-594677833) before being applied. Unsure how we deployed before without the headless service

### Tested

Deployed

### Related issues

- Fixes #7342 

### Backwards compatibility

Because some of the changes to the statefulset are not able to be applied in a normal upgrade, I had to delete the statefulset very quickly and then deploy

### CODE REVIEW

File: packages/helm-charts/eksportisto/templates/statefulset.yaml [hunk 1]
```diff
@@ -1,14 +1,28 @@
+apiVersion: v1
```
Comment (bowd) [inline packages/helm-charts/eksportisto/templates/statefulset.yaml:1]: Will the fact that the service is added here have an impact on targeting the prev deployment with suffix 8?

**Review by bowd — COMMENTED** [review]
LGTM

File: packages/helm-charts/eksportisto/templates/statefulset.yaml (hunk 1 above)
Comment (tkporter) [reply packages/helm-charts/eksportisto/templates/statefulset.yaml:1]: I believe suffix 8 is an old version that is unlikely to be upgraded, and is only still up while we move to suffix 11. But in the meantime I've left suffix 8 untouched. @yorhodes so you're aware if you ever need to upgrade suffix 8

Comment (nambrot) [reply packages/helm-charts/eksportisto/templates/statefulset.yaml:1]: What is this service used for?

**Review by nambrot — APPROVED** [review]
I'd let @yorhodes chime in as he's been closer to the deployment, but lgtm

**Pull request merged**